projects/ngrx.io/content/guide/effects/testing.md

```diff
@@ -186,6 +188,12 @@ effects.getAll$.subscribe(action => {
 });
 </code-example>
 
+<div class="alert is-important">
```

Comment:
What about adding this note beneath the title `With `ReplaySubject`` ?
We could also clarify/update the test by using the `done` callback.

Reply:
Is `done` specific to Jest though?

Reply:
AFAIK most of the test runners have them